token/invocation/schema.go

```diff
@@ -46,26 +48,36 @@ var _ envelope.Tokener = (*tokenPayloadModel)(nil)
 
 // TODO
 type tokenPayloadModel struct {
-	// Issuer DID (sender)
+	// The DID of the Invoker
 	Iss string
-	// Audience DID (receiver)
-	Aud string
-	// Principal that the chain is about (the Subject)
-	// optional: can be nil
-	Sub *string
+	// The DID of Subject being invoked
+	Sub string
+	// The DID of the intended Executor if different from the Subject
+	Aud *string
 
-	// The Command to eventually invoke
+	// The Command
 	Cmd string
-
-	// A unique, random nonce
-	Nonce []byte
+	// The Command's Arguments
+	Args struct {
```

Comment:
smells like that could have the Meta treatment, with a dedicated struct

Reply:
It is EXACTLY like `meta.Meta` and the entire topic of conversation on the "Generalize `meta.Meta` 🧵 "